src/controller/api/newsAPIController.js

```diff
@@ -22,7 +22,10 @@ sources.forEach(async (source) => {
         title: returned.title,
         url: source.base + returned.url,
         source: source.name,
-        image: returned.image ?? defaultImage,
+        image:
+          !!returned.image && !!/^((http|https):\/\/)/.test(returned.image)
```

Comment:
This does work, but it would be more simple (I think) to just use **only** the `RegExp.test(returned.image)` here as the check, because;
- Casting to Boolean isn't necessary as the method returns a Boolean anyway
- No need to check it exists as well. When it doesn't exist it returns `undefined` which fails the "starts with http|https" test right away.

e.g.
```js
        image:
          /^((http|https):\/\/)/.test(returned.image)
          ? returned.image
          : defaultImage,
```